src/redux/reducers/pageSlice.ts

```diff
@@ -0,0 +1,26 @@
+import { actionLiterals } from './actionLiterals';
+
+interface pageReducerState {
+    currentPage: string
+}
+
+interface pageReducerAction {
+    type: string,
+    payload?: string
```

Comment:
The payload type should match the type of the action. This should also be changed to the respective enum (as noted in the other file).